AppointEase/AppointEase.AspNetCore/Controllers/AuthController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using AppointEase.Application.Contracts.Identity;
+using AppointEase.Application.Contracts.Interfaces;
+using AppointEase.Application.Contracts.Models;
+using AppointEase.Application.Contracts.Models.DbModels;
+using AppointEase.Application.Contracts.ModelsDto;
+using AppointEase.Application.Services;
+using AppointEase.Data.Repositories;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace AppointEase.AspNetCore.Controllers
+{
+    public class AuthController : Controller
+    {
+
+        private readonly IPatientService _patientService;
+
+        public AuthController(IPatientService patientService)
+        {
+           this._patientService = patientService;   
+        }
+
+
+        [HttpPost("CreatePatient")]
+        public async Task<IActionResult> CreatePatient([FromBody] PatientRequest patientRequest,string Role)
+        {
+
+            if (!ModelState.IsValid)
```

Comment:
I agree with this approach, if the model is not valid we return bad request and we don't call anything else, but, maybe it's better to validate it through `fluentvalidation`